os/se/ameba/security_ameba_wrapper.c

```diff
@@ -714,7 +753,7 @@ int ameba_hal_get_hash(hal_hash_type mode, hal_data *input, hal_data *hash)
 		return HAL_NOT_SUPPORTED;
 	}
 
-	len = strlen((char*) output);
+	len = strlen((char*)output);
```

Comment:
Please add space between `char` and `*`.

Reply:
Modified.